paasta_tools/api/views/instance.py

```diff
@@ -136,6 +136,7 @@ def instance_status(request):
     instance = request.swagger_data.get("instance")
     verbose = request.swagger_data.get("verbose") or 0
     use_new = request.swagger_data.get("new") or False
+    all_namespaces = request.swagger_data.get("all_namespaces") or False
```

Comment:
this could be 
```suggestion
    all_namespaces = request.swagger_data.get("all_namespaces", False)
```

but this is consistent with some of the other fields in this file so it's fine to keep as-is :)